warehouse/ingest-core/src/main/java/datawave/iterators/PropogatingIterator.java

```diff
@@ -319,11 +331,49 @@ public boolean validateOptions(Map<String,String> options) {
         
         PropogatingCombiner propAgg = null;
         
+        Map<String,Map<String,String>> allAggOptions = new HashedMap();
+        // make a first pass of the options to find any aggregator options and build them up for each aggregator
         for (Entry<String,String> familyOption : options.entrySet()) {
+            String option = familyOption.getKey();
+            // if this property specifies aggregator options skip
+            if (option.contains(AGGREGATOR_OPTS)) {
+                // get the aggregator name
+                String[] splits = option.split("\\.");
+                String aggName = splits[0];
+                if (splits.length < 3) {
+                    throw new IllegalArgumentException("When specifying options for an aggregator specify <cf>." + AGGREGATOR_OPTS + "<option>");
```

Comment:
instead of <option>, use <CombinerOptionName> since that is what was used in the describeOptions method.  Also, I guess this means we can't have any `.` characters in our combiner option names.